test_project/tests/integration_tests/test_admin/test_export/test_admin_class.py

```diff
@@ -81,7 +81,7 @@ def test_get_fieldsets_by_export_job_status(
     mocker.patch(
         "import_export_extensions.models.ExportJob.export_data",
     )
-    job: ExportJob = ArtistExportJobFactory()
+    job: ExportJob = ArtistExportJobFactory.create()
```

Comment:
```suggestion
    job = ArtistExportJobFactory.create()
```
No need for annotations

Reply:
Done